event_sale_reserve_registration/models/sale_order.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def do_registrations(self):
```

Comment:
`do_registrations` seems too generic for a model like `sale.order`.
I'd go for something like this

```suggestion
    def update_event_registrations(self):
```